truffle.js

```diff
@@ -110,7 +110,8 @@ module.exports = {
       currency: 'USD',
       gasPrice: 1,
       showTimeSpent: true
-    }
+    },
+    enableTimeouts: false
```

Comment:
What is this option for?

Reply:
Tests with `eth-gas-reporter` take more time and fail. This option is for disabling timeouts